src/main/java/seedu/address/logic/Messages.java

```diff
@@ -55,12 +55,14 @@ public static String format(Person person) {
     /**
      * Formats the {@code transaction} for display to the user.
      */
-    public static String format(Transaction transaction) {
+    public static String format(Transaction transaction, boolean includeTimestamp) {
```

Comment:
Just to check, was this added to pass unit/integration tests?

Reply:
Currently there are no tests for this class yet